lib/screens/district/district_screen.dart

```diff
@@ -13,16 +17,114 @@ class DistrictScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    CambodiaGeography geo = CambodiaGeography.instance;
+    List<TbCommuneModel> communes = geo.communesSearch(districtCode: district.code.toString());
+    String appBarTitle = (district.type == "SROK" ? 'ស្រុក' : 'ខណ្ឌ') + district.khmer.toString();
     return Scaffold(
       backgroundColor: Theme.of(context).colorScheme.background,
       body: CustomScrollView(
         slivers: [
           MorphingSliverAppBar(
             forceElevated: true,
-            title: CgAppBarTitle(title: district.khmer.toString()),
+            centerTitle: true,
```

Comment:
let us use default `centerTitle`, since it help improve animation when page transition.